integration-tests/mock-grpc-server/main.go

```diff
@@ -49,9 +50,9 @@ func (s *signalServer) PushSignals(stream sensorAPI.SignalService_PushSignalsSer
 			processSignal = signal.GetSignal().GetProcessSignal()
 		}
 
-		processInfo := fmt.Sprintf("%s:%s:%d:%d", processSignal.GetName(), processSignal.GetExecFilePath(), processSignal.GetUid(), processSignal.GetGid())
+		processInfo := fmt.Sprintf("%s:%s:%d:%d:%d:%s", processSignal.GetName(), processSignal.GetExecFilePath(), processSignal.GetUid(), processSignal.GetGid(), processSignal.GetPid(), processSignal.GetArgs())
```

Comment:
I just realized while looking at [this check](https://github.com/stackrox/collector/blob/f5964cbc35478f957aa800476b66aa53fd3df5cb/integration-tests/process.go#L19-L21) that if any of the fields has a `:` this simple serialization would completely break. For now, keeping the arguments as the last part should work fine, but we might want to look into using a proper serializer or even encoding the string parts of the data as base64.